src/components/ShareStructure/ShareStructure.vue

```diff
@@ -522,7 +521,6 @@ export default class ShareStructure extends Mixins(ShareMixin) {
   // Initialize data sets
   mounted () {
     this.updateOriginalIA()
```

Comment:
FYI, instead of updating the local variable here, adding "immediate" to the watch might do the same thing (ie, fire the watch method when this component is initialized).